install_pkgs.R

```diff
@@ -0,0 +1,27 @@
+# Create user library and install devtools package
+dir.create(Sys.getenv('R_LIBS_USER'), recursive = TRUE);
+.libPaths(Sys.getenv('R_LIBS_USER'));
+
+# Install devtools and its dependencies
+install.packages('usethis');
+install.packages('covr');
+install.packages('httr');
+install.packages('rversions');
+install.packages('devtools');
```

Comment:
I haven't tested this, but you could try replacing

```R
install.packages('usethis');
install.packages('covr');
install.packages('httr');
install.packages('rversions');
install.packages('devtools');
```

with 

```R
install.packages('devtools', dependencies=TRUE);
```

And it _should_ handle these automatic installs.